tests/framework/utils/k8s_helper.go

```diff
@@ -644,3 +646,43 @@ func (k8sh *K8sHelper) IsRookInstalled() bool {
 
 	return false
 }
+
+//GetRookLogs captures logs from specified rook pod and writes it to specified file
+func (k8sh *K8sHelper) GetRookLogs(podAppName string, namespace string, filePath string, testName string) {
+	lines := int64(2000)
+	logOpts := &v1.PodLogOptions{TailLines: &lines}
+	listOpts := metav1.ListOptions{LabelSelector: "app=" + podAppName}
+
+	podList, err := k8sh.Clientset.Pods(namespace).List(listOpts)
+	if err != nil {
+		logger.Errorf("Cannot get logs for app : %v in namespace %v", podAppName, namespace)
+		return
+	}
+	podName := podList.Items[0].Name
+	logger.Infof("getting logs for pod : %v", podName)
+
+	res := k8sh.Clientset.Pods(namespace).GetLogs(podName, logOpts).Do()
+	rawData, err := res.Raw()
+	if err != nil {
+		logger.Errorf("Cannot get logs for app : %v in namespace %v, err: %v", podAppName, namespace, err)
+	}
+	dir, _ := os.Getwd()
+	fpath := path.Join(dir, filePath)
```

Comment:
I am going to clean this up, It looks complex because I was testing  support for  generating files when tests are run from jenkins or from localhost i.e. test run from test binary or go test or make test